### PR TITLE
Ask for confirmation before deploying to prod. Fix #13.

### LINKED ISSUE #13: Ask for confirmation when deploying to production
Foot-in-mouth protection. e.g. I don't want to accidentally cause a production deploy because I was too trigger happy selecting a command from my shell history.

Could provide a command line flag to bypass asking for confirmation, if desired. But I think it should be safe by default.

### PR DESCRIPTION
**Pull request by benguillet**

### CODE REVIEW

File: deploy.go [hunk 1]
```diff
@@ -182,6 +183,15 @@ func newDeploymentRequest(c *cli.Context) (*github.DeploymentRequest, error) {
 		return nil, fmt.Errorf("--env flag is required")
 	}
 
+	if env == "production" {
```
Comment (ejholmes) [inline deploy.go]: Can we move this into a named string slice?

``` go
var ProtectedEnvironments = []string{"production"}
```

File: deploy.go [hunk 2]
```diff
@@ -182,6 +183,15 @@ func newDeploymentRequest(c *cli.Context) (*github.DeploymentRequest, error) {
...
+		reader := bufio.NewReader(os.Stdin)
```
Comment (ejholmes) [inline deploy.go]: Maybe move this to an askYN func:

``` go
askYN(prompt string) bool {
}
```

File: deploy.go [hunk 3]
```diff
@@ -341,3 +358,14 @@ func SplitRepo(nwo, defaultOrg string) (owner string, repo string, err error) {
 
 	return
 }
+
+func askYN(prompt string) bool {
+	r := bufio.NewReader(os.Stdin)
+	fmt.Printf("%s (y/N)\n", prompt)
+	a, _ := r.ReadString('\n')
+	if strings.ToUpper(a) == "N\n" {
```
Comment (quentin-) [inline deploy.go]: Shouldn't unknown input not proceed to deploy? why not this?

``` go
return strings.ToUpper(a) == "Y\n" 
```

**Comment by ejholmes** [comment]
Cool! lgtm :+1:

**Pull request merged**